<!DOCTYPE html>
<html lang="en">
  <head><meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta http-equiv="X-UA-Compatible" content="ie=edge">
<link href="https://fonts.googleapis.com/css?family=Merriweather:300|Raleway:400,700" rel="stylesheet">
<link rel="stylesheet" href="/assets/css/style.css">
<title>Publications</title>
<!-- Mathjax Support -->
<script type="text/javascript" async
  src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-MML-AM_CHTML">
</script>
<!-- Begin Jekyll SEO tag v2.6.1 -->
<title>Publications | My playground for data science</title>
<meta name="generator" content="Jekyll v4.0.1" />
<meta property="og:title" content="Publications" />
<meta name="author" content="Hsuan-Chun Lin" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="Write an awesome description for your new site here. You can edit this line in _config.yml. It will appear in your document head meta (for Google search results) and in your feed.xml site description." />
<meta property="og:description" content="Write an awesome description for your new site here. You can edit this line in _config.yml. It will appear in your document head meta (for Google search results) and in your feed.xml site description." />
<link rel="canonical" href="http://localhost:4000/publications/" />
<meta property="og:url" content="http://localhost:4000/publications/" />
<meta property="og:site_name" content="My playground for data science" />
<script type="application/ld+json">
{"headline":"Publications","url":"http://localhost:4000/publications/","description":"Write an awesome description for your new site here. You can edit this line in _config.yml. It will appear in your document head meta (for Google search results) and in your feed.xml site description.","author":{"@type":"Person","name":"Hsuan-Chun Lin"},"@type":"WebPage","@context":"https://schema.org"}</script>
<!-- End Jekyll SEO tag -->

</head>
  <body>
    <main class="container">
      <section class="about">
        <a href="/"><img src="/assets/portfolio.png" alt="Hsuan-Chun Lin"></a>
        <h2 id="title">
          <a href="/">Hsuan-Chun Lin</a>
        </h2>
        <p class="tagline">Computational Biology | Biochemist | Machine learning | A happy self-learner</p>
        <ul class="social"><a href="https://github.com/hsuanchunlin">
              <li>
                <i class="icon-github-circled"></i>
              </li>
            </a></ul><nav class="navigation">
            <ul>
              
                <li>
                  <a href="/publications/">Publications</a>
                </li>
              
                <li>
                  <a href=""></a>
                </li>
              
            </ul>
          </nav><p>&copy;
          2024</p><div>
              <p>Dark Mode
                <i class="icon-moon"></i>
                <label class="switch"> 
                  <input type="checkbox" id="dark-mode-toggle"> 
                  <span class="slider round" onclick="toggleDarkMode()"></span>
                </label>
              </p>
            </div>
            <script type="text/javascript" src="/assets/js/darkmode.js"></script></section>
      <section class="content">
        <div class="post-container">
    <div class="post">
        <h1 id="publications">Publications</h1>

<ol>
  <li>
    <p>A dissociative transition state in Hepatitis Delta Virus ribozyme catalysis. <br /> 
Benjamin Weissman, Şölen Ekesan, <strong>Hsuan-Chun Lin</strong>, Shahbaz Gardezi, Nansheng Li, Timothy J. Giese, Erika McCarthy, Michael E Harris, Darrin M York, and Joseph A Piccirilli,   <strong><em>J. Am. Chem. Soc</em></strong> 2023.</p>
  </li>
  <li>
    <p>Rules of RNA specificity of hnRNP A1 revealed by global and quantitative analysis of its affinity distribution.<br />
Jain, N.; <strong>Lin, H. C.</strong> ; Morgan, C. E.; Harris, M. E.; Tolbert, B. S.<br />
N.J. and H.-C.L. contributed equally to this work.<br />
<strong><em>Proc Natl Acad Sci USA</em></strong>  2017.</p>
  </li>
  <li>
    <p>Analysis of the RNA Binding Specificity Landscape of C5 Protein Reveals Structure and Sequence Preferences that Direct RNase P Specificity.<br />
<strong>Lin, H. C.</strong>; Zhao, J.; Niland, C. N.; Tran, B.; Jankowsky, E.; Harris, M. E.<br />
<strong><em>Cell Chem Biol.</em></strong> 2016, 23 (10), 1271-1281.</p>
  </li>
  <li>
    <p>Determination of relative rate constants for in vitro RNA processing reactions by internal competition.<br />
<strong>Lin, H. C.</strong>, Yandek, L.E., Gjermeni, I. &amp; Harris, M.E.<br />
<strong><em>Anal. Biochem.</em></strong> 467, 54-61 (2014).</p>
  </li>
  <li>
    <p>Determination of the Specificity Landscape for Ribonuclease P Processing of Precursor tRNA 5’ Leader Sequences.<br />
Niland, C. N.; Zhao, J.; <strong>Lin, H. C.</strong>; Anderson, D. R.; Jankowsky, E.; Harris, M.E.<br />
<strong><em>ACS chemical biology</em></strong>  2016, 11 (8), 2285-92.</p>
  </li>
  <li>
    <p>Alternative substrate kinetics of Escherichia coli ribonuclease P: determination of relative rate constants by internal competition.<br />
Yandek, L.E., <strong>Lin, H.C.</strong> ,&amp; Harris, M.E.<br />
<strong><em>J. Biol. Chem.</em></strong> 288, 8342-8354 (2013).</p>
  </li>
  <li>
    <p>Human plasminogen kringle 1-5 reduces atherosclerosis and neointima formation in mice by suppressing the inflammatory signaling pathway.<br />
Chang, P.C., Wu, H.L., <strong>Lin, H.C.</strong>, Wang, K.C. &amp; Shi, G.Y.<br />
<strong><em>J Thromb Haemost</em></strong> 8, 194-201 (2010).</p>
  </li>
  <li>
    <p>Development and Application of High Throughput Methods for Interrogating RNA Binding Specificity.<br />
Lin, H. C.<br />
Case Western Reserve University (2017).<br />
Ph.D. Dissertation.</p>
  </li>
  <li>
    <p>Study the Function of Kringle Proteins in Atherosclerosis.<br />
Lin, H. C.<br />
National Cheng Kung University Library, Taiwan (2007)<br />
Master Thesis.</p>
  </li>
</ol>

<h1 id="conference-talks">Conference Talks</h1>

<ol>
  <li>
    <p>Kinetic isotope effects on catalysis by the HDV ribozyme-precise determination of isotope ratios using electrospray ionization time-of-flight mass spectrometry<br />
<strong>Hsuan-Chun Lin</strong>, Benjamin Weissman, Syed Shahbaz Gardezi, Vernon Anderson, Darrin York, Joseph Piccirilli, Michael Harris<br />
2018 ACS National meeting New Orleans, March 22-26</p>
  </li>
  <li>
    <p>Next-generation tools for RNA enzymology: Determination of rate and equilibrium constants for large populations of RNA substrate variants using high throughput sequencing<br />
<strong>Hsuan-Chun Lin</strong><br />
2014 Rustbelt RNA meeting Pittsburgh, October 17-18</p>
  </li>
</ol>

<h1 id="conference-posters">Conference Posters</h1>
<ol>
  <li><strong>Hsuan-Chun Lin</strong>, Jing Zhao, Courtney Niland, Brandon Tran, Eckhard Jankowsky and Michael E Harris, Global analyses of RNA binding specificity reveal inde- pendent contributions of sequence and structure. Rustbelt RNA Meeting 2015, Cleveland, Oct 23-24, 2015</li>
  <li><strong>Hsuan-Chun Lin</strong>, Courtney Niland, Jing Zhao, Eckhard Jankowsky and Michael E. Harris, Next-generation tools for RNA enzymology: Determination of rate and equilibrium constants for large populations of RNA substrate variants using high throughput sequencing., 20th Annual Meeting of the RNA Society, 2015</li>
  <li><strong>Hsuan-Chun Lin</strong>, Courtney Niland, Jing Zhao, Eckhard Jankowsky and Michael E. Harris, Next-generation tools for RNA enzymology: Determination of rate and equilibrium constants for large populations of RNA substrate variants using high throughput sequencing., 24th Enzyme Mechanisms Conference, 2015</li>
  <li><strong>Hsuan-Chun Lin</strong>, Ulf-Peter Guenther, Courtney Niland, Vernon Anderson, Eckhard Jankowsky, Michael E. Harris, Next-generation Tools for RNA Enzymology: Determination of Rate and Equilibrium Constants for Large Populations of RNA Substrate Variants Using High Throughput Sequencing., 36th Steenbock Symposium May 22-24, 2014</li>
  <li><strong>Hsuan-Chun Lin</strong>, Ulf-Peter Guenther, Courtney Niland, Vernon Anderson, Eckhard Jankowsky, Michael E. Harris, Next-generation tools for RNA enzymology: Determination of rate and equilibrium constants for large populations of RNA substrate variants using high throughput sequencing., Rustbelt RNA Meeting 2013, Cleveland, Oct 18-19, 2013</li>
  <li><strong>HC Lin</strong>, I Gjermeni, E Jankowsky, M Harris , Determination of the species specific interactions between ribonuclease P protein and substrate RNAs by high-throughput sequencing kinetics (HTS-KIN), 245th ACS National Meeting &amp; Exposition, April 7-11, 2013, New Orleans, Louisiana</li>
  <li>Lin, RW., Wang, YH., Lai, SC., <strong>Lin, HC.</strong>, Chang, JK., Ho, ML., Wang, GJ., Microporation-based Non-viral Gene Transfer BMP-2 Enhances Repair of Crit- ical Size Bone Defects. Orthopaedic Research Society. (2011)</li>
  <li>Lin, RW., Wang, YH., Chu, HC., Lai, SC., <strong>Lin, HC.</strong>, Chang, JK., Ho, ML., Wang, GJ. Bone regeneration in critical size defect by hBMSCs mediated BMP- 2 non-viral gene transfer. Orthopaedic Research Society, New Orleans, Louisiana, USA. Mar 6- Mar 9. Abstract ID: ORS2010-2642. (2010)</li>
</ol>

    </div>
</div>
      </section>
    </main></body>
</html>
